sdk/eventhub/eventhubs-checkpointstore-blob/samples/receiveEventsUsingCheckpointStore.ts

```diff
@@ -0,0 +1,77 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how to use the EventHubConsumerClient to process events from all partitions
+  of a consumer group in an Event Hubs instance. It also demonstrates the process of checkpointing an event
+  which helps new instances of your application that may have spun up for scaling or for crash recovery.
+
+  You will see the use of a Partition Manager which is crucial to balance the load of processing events
+  across multiple instances of your application.
+
+  If your Event Hub instance doesn't have any events, then please run "sendEvents.ts" sample
```

Comment:
But there is no sendEvents sample in this project!!

Reply:
True! I made it an HTTP link for now. I'd hate to copy over yet another sample file..